README.md

```diff
@@ -37,6 +37,10 @@ fn code  dig size hash digest
 sha1     4 bytes  4 byte sha1 digest
 ```
 
+The standard encoding is base-58, using the [Bitcoin character set](https://en.bitcoin.it/wiki/Base58Check_encoding#Base58_symbol_chart).
```

Comment:
This is not part of multihash per-se, and weill be rolled out into multibase.

the fact that IPFS uses a base58 encoded multihash is a separate story.